.editorconfig

```diff
@@ -0,0 +1,26 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[Makefile]
+indent_size = 4
+indent_style = tab
```

Comment:
Why mix tabs and spaces in the same codebase? 😕

Reply:
AFAIK, except from GNU Make 3.82+, Makefile files are expected to use tabs.

Reply:
Okay!